docs/sql-guide/functions/function-vector-distances.md

```diff
@@ -1,24 +1,35 @@
 ---
-title: EUCLIDEAN_DISTANCE()
+title: vector distance functions
 layout: default
 parent: Functions
 grand_parent: SQL guide
 ---
-# EUCLIDEAN_DISTANCE() function
+# Vector distance functions
 
-`EUCLIDEAN_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
-The Euclidean (or Euclidian) distance between two vectors is the square root of the sum of the squares
-the pairwise differences between the vectors. This is the distance metric most commonly used when talking
-about distances between locations on a plane.
+Two functions are available to measure the difference between two vectors. They have
+the same syntax, and take the same arguments, but compute different distance metrics.
+
+Both cosine and euclidean distance metrics take two vectors as inputs. Vectors can be
+either vector literals, such as `[0.0, 1.0]` or vector expressions, such as a vector
+column in a table. The vectors must have the same number of dimensions.
 
 ## Syntax
 
 ```sql
+COSINE_DISTANCE (<vector1>, <vector2>)
 EUCLIDEAN_DISTANCE (<vector1>, <vector2>)
 ```
 
-Vectors can be either vector literals, such as `[0.0, 1.0]` or vector expressions, such as a vector
-column in a table. The vectors must have the same number of dimensions.
+## Semantics
```

Comment:
This goes after the arguments, and usually as a h3 under "Additional information"